apps/fhir/bluebutton/utils.py

```diff
@@ -16,9 +19,9 @@
 
 from apps.wellknown.views import (base_issuer, build_endpoint_info)
 from .models import Crosswalk, Fhir_Response
+from apps.logging.request_logger import HHS_SERVER_LOGNAME_FMT
 
-logger = logging.getLogger('hhs_server.%s' % __name__)
-logger_perf = logging.getLogger('performance')
+logger = logging.getLogger(HHS_SERVER_LOGNAME_FMT.format(__name__))
```

Comment:
@JFU-GIT In this imports section, LGTM is alerting with the following. Can this be refactored to pass these alerts?

NEWModule 'apps.logging.request_logger' is imported with both 'import' and 'import from'